test/conformance/source_crd_rbac_test.go

```diff
@@ -0,0 +1,30 @@
+// +build e2e
+
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conformance
+
+import (
+	"testing"
+
+	srchelpers "knative.dev/eventing/test/conformance/helpers/sources"
+	testlib "knative.dev/eventing/test/lib"
+)
+
+func TestSourceCRDRBAC(t *testing.T) {
```

Comment:
Maybe add a high level comment about what this test validates so that one doesn't have to drill into the test below?

Reply:
Looking at all the test files at this level, none of them seem to have comments. So to keep things consistent, I'm adding a high level comment for the test on top of https://github.com/knative/eventing/pull/4787/files#diff-9aedb72c94aebc18f79118d28c71dd67bb52c96c355b82ed237058aede726adcR36. Is that ok?